mxcube3/ui/containers/BeamlineSetupContainer.jsx

```diff
@@ -269,12 +271,10 @@ class BeamlineSetupContainer extends React.Component {
               </Col>
               {this.createActuatorComponent()}
               <Col sm={2} className="pull-right">
-                <LabeledValue
-                  suffix=""
-                  name="Sample changer"
-                  value={this.props.sampleChanger.state}
-                  level={this.props.sampleChanger.state === 'READY' ? 'info' : 'warning'}
-                  look={'vertical'}
+                <SampleChangerSwitch
```

Comment:
Does this also show the state of the sample changer or only if its on or off ?

Reply:
It shows state, same functionality as with the shutters
![image](https://user-images.githubusercontent.com/12758327/55721904-e9c6ce00-5a04-11e9-8bd5-27c05b5a5bef.png)

Reply:
Ok, great :)